NEWS

```diff
@@ -5,18 +5,22 @@ http://pysam.readthedocs.io/en/latest/release.html
 Release notes
 =============
 
-Release 1.0.0?
-==============
-
-* python 2.7 support has been removed. The minimum python version is now 3.2 with
-  versions 3.6-3.11 tested.
-
 Release 0.21.0
 ==============
 
 This release wraps htslib/bcftools version 1.17 and samtools version 1.17.
 
-* [
+Pysam is now compatible with 3.11. We have removed python 2.x
+support. Pysam is tested with python versions 3.7 to 3.11.
+
+* [#1175] VariantHeader.new_record: set start/stop before alleles
+* [#1173] Add multiple build improvements in htscodecs on multi-arch macOS
+* [#1148] Ignore CIGAR-less reads in find_introns.
+* [#1172] Add new `samtools cram-size` and `samtools reset` commands
+* [#1168] Add a minimal pyproject.toml for PEP517.
+* [#1158] Fix type hints and add FastqProxy type hints.
+* [#1147] Py3.11 compatibility, get shared object suffix from EXT_SUFFIX.
+* [#1143] Add mypy symbols for samtools and bcftools.
```

Comment:
These will be largely covered by the build improvements bit you mentioned you'll add to the GitHub release message, but as there are issue numbers for them that people might look for, we might add brief notes to NEWS too:

```
[#1155] Fix pysam.index() when using recent `samtools index` options.
[#1151] Test suite py3.11 compatibility, work around failing test case.
[#1149] MacOS universal build compatibility.
[#1146] Fix build when CFLAGS/etc environment variables are set.
```

Reply:
Thanks, added!